test/performance/pacers.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
May be in a different PR, but perhaps `knative.dev/pkg` is a better place (or even upstream it).

Reply:
Agree, I'll do it in one follow-up PR.